tests/OmniSharp.Roslyn.CSharp.Tests/CodeActionsV2Facts.cs

```diff
@@ -124,50 +124,54 @@ public void Whatever()
-                ("Fully qualify 'Console' -> Internal.Console", CodeActionKind.QuickFix),
-                ("Fully qualify 'Console' -> System.Console", CodeActionKind.QuickFix),
+                    ("using Internal;", CodeActionKind.QuickFix),
+                    ("Fully qualify 'Console' -> Internal.Console", CodeActionKind.QuickFix),
+                    ("Fully qualify 'Console' -> System.Console", CodeActionKind.QuickFix),
 #else
-                ("System.Console", CodeActionKind.QuickFix),
+                    ("System.Console", CodeActionKind.QuickFix),
 #endif
-                ("Generate variable 'Console' -> Generate property 'Console'", CodeActionKind.QuickFix),
-                ("Generate variable 'Console' -> Generate field 'Console'", CodeActionKind.QuickFix),
-                ("Generate variable 'Console' -> Generate read-only field 'Console'", CodeActionKind.QuickFix),
-                ("Generate variable 'Console' -> Generate local 'Console'", CodeActionKind.QuickFix),
-                ("Generate variable 'Console' -> Generate parameter 'Console'", CodeActionKind.QuickFix),
-                ("Generate type 'Console' -> Generate class 'Console' in new file", CodeActionKind.QuickFix),
-                ("Generate type 'Console' -> Generate class 'Console'", CodeActionKind.QuickFix),
-                ("Generate type 'Console' -> Generate nested class 'Console'", CodeActionKind.QuickFix),
-                ("Extract local function", CodeActionKind.RefactorExtract),
-                ("Extract method", CodeActionKind.RefactorExtract),
-                ("Introduce local for 'Console.Write(\"should be using System;\")'", CodeActionKind.Refactor)
-            } : new List<(string Name, string CodeActionKind)>
-            {
-                ("using System;", CodeActionKind.QuickFix),
+                    ("Generate variable 'Console' -> Generate property 'Console'", CodeActionKind.QuickFix),
+                    ("Generate variable 'Console' -> Generate field 'Console'", CodeActionKind.QuickFix),
+                    ("Generate variable 'Console' -> Generate read-only field 'Console'", CodeActionKind.QuickFix),
+                    ("Generate variable 'Console' -> Generate local 'Console'", CodeActionKind.QuickFix),
+                    ("Generate variable 'Console' -> Generate parameter 'Console'", CodeActionKind.QuickFix),
+                    ("Generate type 'Console' -> Generate class 'Console' in new file", CodeActionKind.QuickFix),
+                    ("Generate type 'Console' -> Generate class 'Console'", CodeActionKind.QuickFix),
+                    ("Generate type 'Console' -> Generate nested class 'Console'", CodeActionKind.QuickFix),
+                    ("Extract local function", CodeActionKind.RefactorExtract),
+                    ("Extract method", CodeActionKind.RefactorExtract),
+                    ("Use expression body for method", CodeActionKind.Refactor),
```

Comment:
Expression body refactoring are now being offered for non-empty selections https://github.com/dotnet/roslyn/pull/76969